plugins/lsp/default.nix

```diff
@@ -225,7 +225,7 @@ in
           ${cfg.preConfig}
 
           local __lspServers = ${helpers.toLuaObject cfg.enabledServers}
-          local __lspOnAttach = function(client, bufnr)
+          M.lspOnAttach = function(client, bufnr)
```

Comment:
Because this is a new practice, maybe note in a comment why we are assigning to the `M` global?